src/Button/Button.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import './style.css';
+
+type ButtonProps = React.ComponentPropsWithoutRef<'button'>;
```

Comment:
Need to discuss.

Reply:
Changed ComponentPropsWithoutRef to ButtonHTMLAttributes.